cms/djangoapps/contentstore/views/tests/test_bulksettings.py

```diff
@@ -211,19 +206,21 @@ def do_values_match(self, setting_types, child, computed_settings):
         """
 
         for setting_type in setting_types:
-            setting= getattr(child, setting_type)
+            setting = getattr(child, setting_type)
 
             if setting_type == 'start' or setting_type == 'due':
                 setting = setting.strftime('%m/%d/%Y')
 
-            self.assertEqual(setting, computed_settings[setting_type],
-                            msg="setting_type" + str(setting_type) + "does not match")
+                self.assertEqual(
+                    setting,
+                    computed_settings[setting_type],
+                    msg="setting_type" + str(setting_type) + "does not match"
```

Comment:
That makes a LOT of sense.  Sorry I didn't notice this, and thanks for showing me how string formatting can help prevent cases like this.